s3/pkg/datastore/yig/crypto/header.go

```diff
@@ -0,0 +1,220 @@
+// Minio Cloud Storage, (C) 2015, 2016, 2017, 2018 Minio, Inc.
```

Comment:
please remove this reference to Minio...also, as in comment above, we can hold off on ALL the 'crypto' related code till we discuss on the 

> data encryption requirement